store/src/main/scala/net/spals/leagur/store/hyperdex/impl/HyperdexStore.scala

```diff
@@ -0,0 +1,41 @@
+package net.spals.leagur.store.hyperdex.impl
+
+import java.util.{Optional, Collections}
+
+import com.netflix.governator.annotations.AutoBindSingleton
+import net.spals.leagur.model.Entity
+import net.spals.leagur.store.Store
+
+import scala.collection.JavaConverters._
+
+/**
+ * Hyperdex implementation of {@link Store}
+ *
+ * @author tkral
+ */
+@AutoBindSingleton(classOf[Store])
+class HyperdexStore extends Store {
```

Comment:
Perhaps we will add an InMemoryStore for testing. I've found that useful when writing some db layered things.